tests/disko-install/default.nix

```diff
@@ -5,6 +5,11 @@ let
   dependencies = [
     self.nixosConfigurations.testmachine.pkgs.stdenv.drvPath
     (self.nixosConfigurations.testmachine.pkgs.closureInfo { rootPaths = [ ]; }).drvPath
+
+    # https://github.com/NixOS/nixpkgs/blob/f2fd33a198a58c4f3d53213f01432e4d88474956/nixos/modules/system/activation/top-level.nix#L342
+    self.nixosConfigurations.your-machine.pkgs.perlPackages.ConfigIniFiles
+    self.nixosConfigurations.your-machine.pkgs.perlPackages.FileSlurp
```

Comment:
Shouldn't this refer to `testmachine` instead?

Reply:
https://github.com/nix-community/disko/commit/a8bdb16b473f457609eb76d0dabb2fcb4e1f64b8